rpcs3/Emu/RSX/gcm_printing.cpp

```diff
@@ -710,20 +710,75 @@ namespace
 	    {NV4097_SET_VERTEX_ATTRIB_INPUT_MASK, "NV4097_SET_VERTEX_ATTRIB_INPUT_MASK"},
 	    {NV4097_SET_VERTEX_ATTRIB_OUTPUT_MASK, "NV4097_SET_VERTEX_ATTRIB_OUTPUT_MASK"},
 	    {NV4097_SET_TRANSFORM_BRANCH_BITS, "NV4097_SET_TRANSFORM_BRANCH_BITS"},
+	    {NV0039_SET_OBJECT, "NV0039_SET_OBJECT"},
+	    {NV0039_SET_CONTEXT_DMA_NOTIFIES, "NV0039_SET_CONTEXT_DMA_NOTIFIES"},
+	    {NV0039_SET_CONTEXT_DMA_BUFFER_IN, "NV0039_SET_CONTEXT_DMA_BUFFER_IN"},
+	    {NV0039_SET_CONTEXT_DMA_BUFFER_OUT, "NV0039_SET_CONTEXT_DMA_BUFFER_OUT"},
+	    {NV0039_OFFSET_IN, "NV0039_OFFSET_IN"},
+	    {NV0039_OFFSET_OUT, "NV0039_OFFSET_OUT"},
+	    {NV0039_PITCH_IN, "NV0039_PITCH_IN"},
+	    {NV0039_PITCH_OUT, "NV0039_PITCH_OUT"},
+	    {NV0039_LINE_LENGTH_IN, "NV0039_LINE_LENGTH_IN"},
+	    {NV0039_LINE_COUNT, "NV0039_LINE_COUNT"},
+	    {NV0039_FORMAT, "NV0039_FORMAT"},
+	    {NV0039_BUFFER_NOTIFY, "NV0039_BUFFER_NOTIFY"},
 	    {NV3062_SET_OBJECT, "NV3062_SET_OBJECT"},
 	    {NV3062_SET_CONTEXT_DMA_NOTIFIES, "NV3062_SET_CONTEXT_DMA_NOTIFIES"},
 	    {NV3062_SET_CONTEXT_DMA_IMAGE_SOURCE, "NV3062_SET_CONTEXT_DMA_IMAGE_SOURCE"},
 	    {NV3062_SET_CONTEXT_DMA_IMAGE_DESTIN, "NV3062_SET_CONTEXT_DMA_IMAGE_DESTIN"},
-	    {NV3062_SET_COLOR_FORMAT, "NV3062_SET_COLOR_FORMAT"}, {NV3062_SET_PITCH, "NV3062_SET_PITCH"},
+	    {NV3062_SET_COLOR_FORMAT, "NV3062_SET_COLOR_FORMAT"},
+		{NV3062_SET_PITCH, "NV3062_SET_PITCH"},
```

Comment:
like here and some other places.

Reply:
Wow, you should go with tabs then. you can fix this if you want. ~~inb4 5000+ 5000- lines changes in pr~~